Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -220,11 +221,32 @@ bool experimentalFeatureEnabled(const String& key, bool defaultValue)
 
 std::optional<bool> WebsiteDataStore::useNetworkLoader()
 {
-#if HAVE(NETWORK_LOADER)
-    return optionalExperimentalFeatureEnabled(WebPreferencesKey::cFNetworkNetworkLoaderEnabledKey(), std::nullopt);
-#else
+#if !HAVE(NETWORK_LOADER)
     return false;
+#else
+
+    [[maybe_unused]] const auto isSafari =
+#if PLATFORM(MAC)
+        MacApplication::isSafari();
+#elif PLATFORM(IOS_FAMILY)
+        WebCore::IOSApplication::isMobileSafari() || WebCore::IOSApplication::isSafariViewService();
+#else
```

Comment:
Should we be checking for `default web browser`, instead of safari specifically? Would we want to use this loader for Mail or any other WebKit clients? What about something like News that loads web content?

Reply:
Great questions, but for now I was told that we only want this for "Safari", and we can include other full web browsers and other applications later.